client/Dockerfile

```diff
@@ -1,19 +1,5 @@
-FROM debian:latest
+FROM node:0.10
```

Comment:
I know David did FROM debian instead of FROM node deliberately. So I guess it would be best if he gave a thumbs-up on this change...